### PR TITLE
fix: remove mayastor pool finalizers check

### PR DESCRIPTION
**Pull request by rohan2794**

### CODE REVIEW

File: src/common/k8stest/test.go [hunk 1]
```diff
@@ -179,13 +179,16 @@ func AfterSuiteCleanup() {
...
 func CheckMsPoolFinalizers() error {
-	err := custom_resources.CheckAllMsPoolFinalizers()
 	logf.Log.Info("Checking pool finalizers", "timeout seconds", e2e_config.GetConfig().MoacSyncTimeoutSeconds)
 	const sleepTime = 5
+	var err error
 	t0 := time.Now()
 	for ix := 0; ix < e2e_config.GetConfig().MoacSyncTimeoutSeconds && err != nil; ix += sleepTime {
 		time.Sleep(sleepTime * time.Second)
 		err = custom_resources.CheckAllMsPoolFinalizers()
+		if err == nil {
```
Comment (chriswldenyer) [inline src/common/k8stest/test.go:189]: Does this not exit the loop anyway because of the test on line 186 (err != nil) ?

**Review by chriswldenyer — COMMENTED** [review]
One question

File: src/common/k8stest/test.go (hunk 1 above)
Comment (rohan2794) [reply src/common/k8stest/test.go]: removed pool finalizer check

**Review by blaisedias — APPROVED** [review]
LGTM

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Comment by rohan2794** [comment]
bors merge

**Pull request merged**